openfermioncirq/experiments/hfvqe/README.md

```diff
@@ -0,0 +1,29 @@
+Hartree-Fock on a superconducting qubit quantum processor
```

Comment:
Since this is a implementation of VQE, shouldn't this be in the  variational (https://github.com/quantumlib/OpenFermion-Cirq/tree/master/openfermioncirq/variational) directory?

Reply:
the current structure of openfermion-cirq will probably change drastically soon.